VERSION

```diff
@@ -1,6 +1,7 @@
 MAJOR=0
 MINOR=1
 PATCH=6
+RC_VERSION=2
```

Comment:
Let's call this `DRAFT`

Reply:
So. `DRAFT=2`, or `DRAFT_VERSION=2`?

Reply:
So, `DRAFT=2`, or `DRAFT_VERSION=2`?

Reply:
I think `DRAFT`

Reply:
Okay. 21905b15bb47c2ab21a8d670d5032cd6b7a46c7b fixes this.